limeade3/urls.py

```diff
@@ -1,12 +1,11 @@
-from django.conf.urls import patterns, include, url
+from django.conf.urls import include, url
 from django.contrib import admin
 from django.views.generic import RedirectView
 admin.autodiscover()
 
-urlpatterns = patterns('',
+urlpatterns = [
 	url(r'^$', RedirectView.as_view(url='/mail/')),
 	url(r'^mail/', include('mail.urls')),
-	url(r'^admin/', include(admin.site.urls)),
+#	url(r'^admin/', include(admin.site.urls)),
```

Comment:
intentional?

Reply:
Missed to remove the commend and remove the `include()`. Fixed in the latest commit